bundled/magic_enum/README.md

```diff
@@ -0,0 +1,250 @@
+```text
```

Comment:
I am not sure whether copying the readme is the correct procedure? The readme text we have here will be out-dated soon, so why not just putting a link to the true readme of magic-enum? A different aspect that I am not sure about is whether we actually have the rights to copy the content of a readme file?

Reply:
when writing this comment, I have not seen that source code is copied as well. So maybe this is the standard procedure ... Then you can probably forget about this comment.

Reply:
Yes, the README is in sync since the source code is also copied. As far as I know, this is OK since the project is under the MIT Licence. As I mentioned in https://github.com/exadg/exadg/pull/264#discussion_r1133288168 I would need some help if we take a different route (with submodules).